conf/local.conf.sample

```diff
@@ -54,6 +54,10 @@ MACHINE ??= "qemux86"
 IMAGE_FEATURES_DEVELOPMENT ?= "debug-tweaks codebench-debug ssh-server-openssh tools-profile"
 EXTRA_IMAGE_FEATURES = "${IMAGE_FEATURES_DEVELOPMENT}"
 
+# Uncomment the following to enable BSP specific patches and configurations for
+# kernel debugging
+# MACHINE_FEATURES_append = " mel-debugging"
```

Comment:
@ahsanhussain, I think it would be a good idea to remove the space between MACHINE_FEATURES and the # just to conform with the rest of the configuration mechanism.

Reply:
@abelal done. It was just my vim gimmicks (nerdcommentor) :p